gprofiler/main.py

```diff
@@ -1048,6 +1078,7 @@ def main() -> None:
             args.duration,
             args.profile_api_version,
             args.profiling_mode,
+            args.pids_to_profile,
```

Comment:
This can also be made kwargs-only if you'd like in this PR :) Can also do in another.